### PR TITLE
What's new updates Aug 2 - 9

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

This pull request updates the [What's New](https://devdocs.magento.com/whats-new.html) page with the documentation changes published since the last update on August 2.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,30 @@ description: |
...
 thread: /whatsnew-feed.xml
-updated: Fri Aug 2 12:05:50 2019
+updated: Fri Aug 9 23:29:50 2019
 entries:
+- description: Added information about how to [define a custom Import Entity](https://devdocs.magento.com/guides/v2.3/ext-best-practices/tutorials/custom-import-entity.html).
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: August 7, 2019
+  link: https://github.com/magento/devdocs/pull/5007
+- description: Updated the label descriptions [devdocs contributions](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs), corrected links, and added link to the [Beginner's Guide](https://github.com/magento/magento2/wiki/Getting-Started).
```
Comment (meker12) [inline _data/whats-new.yml]:
```suggestion
- description: Updated the label descriptions for [devdocs contributions](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs), corrected links, and added link to the [Beginner's Guide](https://github.com/magento/magento2/wiki/Getting-Started).
```

File: _data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,30 @@ description: |
...
 thread: /whatsnew-feed.xml
-updated: Fri Aug 2 12:05:50 2019
+updated: Fri Aug 9 23:29:50 2019
 entries:
+- description: Added information about how to [define a custom Import Entity](https://devdocs.magento.com/guides/v2.3/ext-best-practices/tutorials/custom-import-entity.html).
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: August 7, 2019
+  link: https://github.com/magento/devdocs/pull/5007
+- description: Updated the label descriptions for [devdocs contributions](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs), corrected links, and added link to the [Beginner's Guide](https://github.com/magento/magento2/wiki/Getting-Started).
```
Comment (shrielenee) [inline _data/whats-new.yml]:
```suggestion
- description: Updated the label descriptions for [DevDocs contributions](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs), corrected links, and added a link to the [Beginner's Guide](https://github.com/magento/magento2/wiki/Getting-Started).
```

File: _data/whats-new.yml [hunk 3]
```diff
@@ -4,8 +4,30 @@ description: |
...
+  type: Major update
+  date: August 5, 2019
+  link: https://github.com/magento/devdocs/pull/4551
+- description: Added a new topic about [enabling database query profiling](https://devdocs.magento.com/guides/v2.3/cloud/project/profile-database-queries.html).
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: August 2, 2019
+  link: https://github.com/magento/devdocs/pull/5111
+- description: Added a new task topic for [setting the cache time-to-live (TTL) for
+    static files](https://devdocs.magento.com/guides/v2.3/cloud/project/set-cache.md)
```
Comment (dshevtsov) [inline _data/whats-new.yml]: the link points to the `.md` file

File: _data/whats-new.yml [hunk 4]
```diff
@@ -4,8 +4,30 @@ description: |
...
+  type: New topic
+  date: August 2, 2019
+  link: https://github.com/magento/devdocs/pull/5111
+- description: Added a new task topic for [setting the cache time-to-live (TTL) for
+    static files](https://devdocs.magento.com/guides/v2.3/cloud/project/set-cache.html)
+    on Magento Commerce (Cloud).
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: August 2, 2019
+  link: https://github.com/magento/devdocs/pull/5109
```
Comment (dshevtsov) [inline _data/whats-new.yml]: The original PR still contains the link to `.md` file:
https://github.com/magento/devdocs/pull/5109

Comment (dshevtsov) [reply _data/whats-new.yml]: `whatsnew` in the PR still contains link to `.md` file

Comment (meker12) [reply _data/whats-new.yml]: image

Comment (dshevtsov) [reply _data/whats-new.yml]: check link in `whatsnew` please

Comment (dshevtsov) [reply _data/whats-new.yml]: @meker12 the link you changed is out of scope for this pull request.
Target content for this PR follows the `whatsnew` tag.

Comment (dshevtsov) [reply _data/whats-new.yml]: Screen Shot 2019-08-12 at 15 52 29

Comment (jeff-matthews) [reply _data/whats-new.yml]: I fixed it

**Pull request merged**